lms/djangoapps/student_account/views.py

```diff
@@ -190,6 +199,62 @@ def password_change_request_handler(request):
         return HttpResponseBadRequest(_("No email address provided."))
 
 
+def enterprise_fields_only(fields):
+    enterprise_exclusions = {
+        'age',
+        'level_of_education',
+        'gender',
+        'goals',
+        'year_of_birth',
+        'mailing_address',
+    }
+    fields = json.loads(fields)
+    fields['fields'] = [field for field in fields['fields'] if field['name'] not in enterprise_exclusions]
+    return json.dumps(fields)
+
+def enterprise_sidebar(request):
+    enterprise_customer = enterprise_customer_for_request(request)
+
+    if not enterprise_customer:
+        return ''
+
+    platform_name = configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
+
+    if enterprise_customer.branding_configuration.logo:
+        enterprise_logo_url = enterprise_customer.branding_configuration.logo.url
+    else:
+        enterprise_logo_url = ''
+
+    if getattr(enterprise_customer.branding_configuration, 'welcome_message', None):
+        branded_welcome_template = enterprise_customer.branding_configuration.welcome_message
+    else:
+        branded_welcome_template = configuration_helpers.get_value(
+            'ENTERPRISE_SPECIFIC_BRANDED_WELCOME_TEMPLATE',
+            settings.ENTERPRISE_SPECIFIC_BRANDED_WELCOME_TEMPLATE
+        )
+
+    branded_welcome_string = branded_welcome_template.format(
+        start_bold='<b>',
+        end_bold='</b>',
+        enterprise_name=enterprise_customer.name,
+        platform_name=platform_name
+    )
+
+    platform_welcome_template = configuration_helpers.get_value(
+        'ENTERPRISE_PLATFORM_WELCOME_TEMPLATE',
+        settings.ENTERPRISE_PLATFORM_WELCOME_TEMPLATE
+    )
+    platform_welcome_string = platform_welcome_template.format(platform_name=platform_name)
+
+    context = {
+        'enterprise_logo_url': enterprise_logo_url,
+        'enterprise_branded_welcome_string': branded_welcome_string,
+        'platform_welcome_string': platform_welcome_string,
+    }
+
+    return render_to_string('student_account/enterprise_sidebar.html', context)
```

Comment:
Is there a reason we would `render_to_string` here instead of just doing a template include of the `enterprise_sidebar.html` template in the `login_and_register.html` template?